itest/lnd_open_channel_test.go

```diff
@@ -594,3 +594,44 @@ func verifyCloseUpdate(chanUpdate *lnrpc.ChannelEventUpdate,
 
 	return nil
 }
+
+// testFundingExpiryBlocksOnPending checks that after an OpenChannel, and
+// before the funding transaction is confirmed, that the FundingExpiryBlocks
+// field of a PendingChannels decreases.
+func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
+	// The default number of confirmations for a channel open might not be
+	// enough for testing, so configure bob to require a few more confs for
+	// a channel to be considered open.
+	alice, bob := ht.Alice, ht.Bob
+	const numConfs = 3
```

Comment:
Alternatively we could use `ht.MineEmptyBlocks` to keep the funding tx from being confirmed.

Reply:
Ah, I wasn't aware that we had that helper function. So I agree, since it makes the test much easier to read and might even be faster than restarting the nodes.
So I think we can just mine 3 empty blocks in the loop below, then mine one "normal" one with the funding transaction and have the same test coverage.